app/merginapi.cpp

```diff
@@ -1229,9 +1259,48 @@ void MerginApi::listProjectsReplyFinished()
   }
 
   r->deleteLater();
+
+  Q_UNUSED( requestId )
+  //TODO: add requestId to signal
   emit listProjectsFinished( mRemoteProjects, mTransactionalStatus, projectCount, requestedPage );
 }
 
+void MerginApi::listProjectsByNameReplyFinished( QString requestId )
```

Comment:
I would keep them now in separate places because I believe their code will not look the same in the end. However, I am adding a todo note to remember this if they would end up the same